admin/spec/features/spree/admin/products/option_types_spec.rb

```
@@ -0,0 +1,72 @@
+      visit spree.new_admin_option_type_path
+      fill_in 'Presentation', with: 'Hat Size'
+      click_button 'Create'
+      expect(page).to have_content('successfully created!')
+
+      fill_in 'Presentation', with: 'Hat Size Updated'
+      within('#page-header') { click_button 'Update' }
+      expect(page).to have_content('successfully updated!')
+    end
+  end
+
+  describe 'manaing option values', js: true do
```

RuboCop (1.75.5)

[convention] Use symbol style for metadata.

(RSpec/MetadataStyle)